app/move/controllers/create/assessment.test.js

```diff
@@ -1,8 +1,8 @@
 const FormController = require('hmpo-form-wizard').Controller
 
-const Controller = require('./new.assessment')
-const referenceDataService = require('../../../common/services/reference-data')
-const referenceDataHelpers = require('../../../common/helpers/reference-data')
+const Controller = require('./assessment')
```

Comment:
Can this become `AssessmentController`?

Reply:
I think that was just when I was keeping the tests a little simple. The test is testing that particular controller and it's not importing multiple controllers so didn't see the need to have a more verbose name.

If we want to rename things like this it might be better as a separate change, what do you think?